gapic/src/main/com/google/gapid/models/Settings.java

```diff
@@ -341,9 +339,8 @@ private static boolean getBoolean(Properties properties, String name, boolean df
     if (value == null) {
       return dflt;
     }
-    return stream(
-        Splitter.on(',').trimResults().omitEmptyStrings().split(value).spliterator(), false)
-        .toArray(l -> new String[l]);
+    return stream(Splitter.on(',').trimResults().omitEmptyStrings().split(value).spliterator(),
```

Comment:
I think the previous version was more readable. Having a line start with "false).toArray(...)" is not exactly nice.